pkg/transform/util/util.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+)
+
+type SecretDef struct {
+	SecretName string
+	DataKey    string
+}
+
+func SecretRefToSecretDef(secretRef *string) (*SecretDef, error) {
```

Comment:
I am not sure about keeping this here, because `util` generally should be something that is generic and can be used in varied number of places.